.github/workflows/ci.yml

```diff
@@ -205,6 +205,12 @@ jobs:
         use-cross: true
         command: build
         args: --target ${{ matrix.target }} --manifest-path "aws/rust-runtime/Cargo.toml" ${{ matrix.aws_excludes }} --workspace
+    - name: Test AWS rust-runtime crates
```

Comment:
Should we run tests for `rust-runtime` crates as well at this point?

Reply:
Yeah, I am working on it!